test/wasi/test-wasi-options-validation.js

```diff
@@ -43,13 +43,16 @@ assert.throws(() => { new WASI({ stderr: 'fhqwhgads' }); },
 
 // Verify that exceptions thrown from the binding layer are handled.
 assert.throws(() => {
-  new WASI({ preopens: { '/sandbox': '__/not/real/path' } });
+  new WASI({ version: 'preview1', preopens: { '/sandbox': '__/not/real/path' } });
 }, { code: 'UVWASI_ENOENT', message: /uvwasi_init/ });
 
 // If version is not a string, it should throw
 assert.throws(() => { new WASI({ version: { x: 'y' } }); },
               { code: 'ERR_INVALID_ARG_TYPE', message: /\bversion\b/ });
 
+// If version is not specificed,  it should throw
```

Comment:
Do we have a test anywhere for an invalid/unsupported version? If not, we could copy this and pass some garbage string as the version.

Reply:
We have those tests. The next two tests following this one validate that the version is a string and that it is not just some random string.